### DISCUSSION TITLE
GraphQL, how to type frontend gql`` ?

### DISCUSSION THREAD

**Discussion by NgyAnthony** [ohmyform/ohmyform#173] [Q&A] [answered]
Hi !

I have a bit of a newbie question on GraphQL (never worked with it).

In the following code (ui/graphql/query/submission.pager.query.ts)

![Capture d’écran 2022-04-08 à 13 51 07](https://user-images.githubusercontent.com/26002874/162431074-925ac0de-0198-4cd2-a432-df2d9b6d2d82.png)

Using IntelliJ Idea, some code is in red and it doesn't find the reference. 
How can I add the reference to the gql schema ? 

Thanks !

**Comment by wodka** [comment, marked as answer]
There is a file called `.graphqlconfig` -> in that you should be able to press "play" next to a valid endpoint - this will then create a `schema.graphql` and with that show validation / autocomplete in gql
